src/applications/personalization/dashboard-2/components/apply-for-benefits/ApplyForBenefits.unit.spec.jsx

```diff
@@ -3,7 +3,7 @@ import { expect } from 'chai';
 
```

Comment:
I will add some more tests to this file to make sure the `BenefitsOfInterest` works